crates/core/src/db/models.rs

```diff
@@ -1677,3 +1677,14 @@ pub struct CurrentMetadataOwner<'a> {
     /// The period of time for which each leader ingests transactions and produces a block.
     pub slot: i64,
 }
+
+/// A row in the `store_auction_houses` table
+#[derive(Debug, Clone, Queryable, Insertable, AsChangeset, QueryableByName)]
+#[diesel(treat_none_as_null = true)]
+#[table_name = "store_auction_houses"]
```

Comment:
Need to run migrate on your local db to get the schema.rb to adjust that will resolve these errors.